federatedscope/autotune/fedex/server.py

```diff
@@ -81,7 +81,8 @@ def __init__(self,
               self).__init__(ID, state, config, data, model, client_num,
                              total_round_num, device, strategy, **kwargs)
 
-        if self._cfg.federate.restore_from != '':
+        if self._cfg.federate.restore_from != '' and os.path.exists(
+                self._cfg.federate.restore_from):
```

Comment:
if it is not am empty string but no such a file, we'd better logger.warn sth.

Reply:
Updated.